.changeset/gentle-vans-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
```

Comment:
Changes to SvelteKit are all internal, `create-svelte` changes because we bump the version that's scaffolded
```suggestion
'create-svelte': patch
```

Reply:
Oops. Yes. Thought that's what I had. Thanks for catching it